submission/persistence-filesystem/src/main/kotlin/ac/uk/ebi/biostd/persistence/filesystem/nfs/NfsFtpService.kt

```diff
@@ -53,5 +55,41 @@ class NfsFtpService(
         return nfsFile
     }
 
-    private fun getPublicFolder(relPath: String): Path = FileUtils.getOrCreateFolder(folderResolver.getPublicSubFolder(relPath), RWXR_XR_X)
+    override suspend fun suppressSubmissionFile(
```

Comment:
I am not the best with names or english but I simple do not relate `supress` remove published files. unrelease, unpublish sounds way better and natural for me. Maybe is worth it ask team about this (as is basically the domain language)

Reply:
I agree with you, I just went with this name because it was the name that was given to the operation of take a public submission and make it private